bot/exts/events/advent_of_code/_helpers.py

```diff
@@ -151,7 +158,7 @@ def _parse_raw_leaderboard_data(raw_leaderboard_data: dict) -> dict:
         # this data to JSON in order to cache it in Redis.
         daily_stats[day] = {"star_one": star_one, "star_two": star_two}
 
-    return {"daily_stats": daily_stats, "leaderboard": sorted_leaderboard}
+        return {"daily_stats": daily_stats, "leaderboard": sorted_leaderboard, 'per_day_and_star': per_day_star_stats}
```

Comment:
Was this intended? This will now return after the first iteration.

Reply:
Absolutely not, the logic was moved around a bit and I did not test daily stats just the `per_day_and_star` stats, so I did not notice. Thank you! 👍